{"uuid": "f2c0ad67-3fa7-44b5-b51f-299c445ea9af", "children": ["dc331ce8-60f6-4d15-8b1d-e731735647a7"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"d:\\anaconda3\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"d:\\anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"d:\\anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"d:\\anaconda3\\lib\\site-packages\\_pytest\\python.py\", line 701, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"d:\\anaconda3\\lib\\site-packages\\_pytest\\python.py\", line 632, in _call_with_optional_argument\n    func(arg)\n  File \"E:\\py-auto\\icdm_Interface\\testcase\\test_icdm_update_treat_plan_byid.py\", line 16, in setup_class\n    ApiGetToken().get_token_success(self.session)\n  File \"E:\\py-auto\\icdm_Interface\\api\\api_get_token.py\", line 60, in get_token_success\n    setting.access_token = response.json().get('access_token')  # 产生数据，并把数据放到setting当中\n  File \"d:\\anaconda3\\lib\\site-packages\\requests\\models.py\", line 900, in json\n    return complexjson.loads(self.text, **kwargs)\n  File \"d:\\anaconda3\\lib\\json\\__init__.py\", line 348, in loads\n    return _default_decoder.decode(s)\n  File \"d:\\anaconda3\\lib\\json\\decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"d:\\anaconda3\\lib\\json\\decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\n"}, "start": 1616390932221, "stop": 1616390932230}], "start": 1616390932221, "stop": 1616390932281}